src/Tizen.WebView/Tizen.WebView/ContextMenu.cs

```diff
@@ -0,0 +1,554 @@
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenuItemEventArgs : EventArgs
+    {
+        private ContextMenuItem _menuItem;
+
+        internal ContextMenuItemEventArgs(IntPtr handle)
+        {
+            _menuItem = new ContextMenuItem(handle);
+        }
+
+        /// <summary>
+        /// Gets the context menu item.
+        /// </summary>
+        /// <returns>The context menu item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItem GetMenuItem()
+        {
+            return _menuItem;
+        }
+
+        internal static ContextMenuItemEventArgs CreateFromSmartEvent(IntPtr data, IntPtr obj, IntPtr info)
+        {
+            return new ContextMenuItemEventArgs(info);
+        }
+    }
+
+    /// <summary>
+    /// This class provides the properties of Context Menu.
+    /// </summary>
+    /// <since_tizen> 6 </since_tizen>
+    public class ContextMenu
+    {
+        private IntPtr _handle;
+
+        internal ContextMenu(IntPtr handle)
+        {
+            _handle = handle;
+        }
+
+        /// <summary>
+        /// Gets the context menu items count.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        public int ItemCount
+        {
+            get
+            {
+                return Interop.ChromiumEwk.ewk_context_menu_item_count(_handle);
+            }
+        }
+
+        /// <summary>
+        /// Gets Nth item of the context menu.
+        /// </summary>
+        /// <param name="n"> To get item at index n.</param>
+        /// <returns>The context menu item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItem GetItemAtIndex(int n)
+        {
+            IntPtr itemHandle = Interop.ChromiumEwk.ewk_context_menu_nth_item_get(_handle, n);
+            return new ContextMenuItem(itemHandle);
+        }
+
+        /// <summary>
+        /// Removes item from the context menu.
+        /// </summary>
+        /// <param name="item"> The context menu item to be removed.</param>
+        /// <since_tizen> 6 </since_tizen>
+        public void RemoveItem(ContextMenuItem item)
+        {
+            IntPtr itemHandle = item.GetHandle();
+            Interop.ChromiumEwk.ewk_context_menu_item_remove(_handle, itemHandle);
+        }
+
+        /// <summary>
+        /// Appends item to the context menu.
+        /// </summary>
+        /// <param name="tag"> The tag of context menu item.</param>
+        /// <param name="title"> The title of context menu item.</param>
+        /// <param name="iconPath"> The path of icon to be set on context menu item.</param>
+        /// <param name="enabled"> if true the context menu item is enabled else false.</param>
+        /// <returns>Appended context menu item.</returns>
+        /// <since_tizen> 6 </since_tizen>
+        public ContextMenuItem AppendItem(ContextMenuItemTag tag, string title, string iconPath, bool enabled)
```

Comment:
What is the use case of this return value? I would expect to have a new value appended to the end of my existing collection (returns void) or receive a copy of the collection with the new element.

Reply:
It implements the functionality to create and add a ContextMenuItem with ContextMenuItemTag argument.
User want to get created ContentMenuItem object. so it return a created object

It is common implementation pattern
* similar case https://github.com/Samsung/TizenFX/blob/14145c440a5b0b22a2e1e5250db0581414dd0e71/src/ElmSharp/ElmSharp/GenList.cs#L554